runtimeconfig/google/cloud/runtimeconfig/_helpers.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Shared helper functions for RuntimeConfig API classes."""
+
+
+def _config_name_from_full_name(full_name):
+    """Extract the config name from a full resource name.
+    
+      >>> _config_name_from_full_name('projects/my-proj/configs/my-config')
+      "my-config"
+    """
+    return full_name.split('/')[3]
+
+def _variable_name_from_full_name(full_name):
```

Comment:
Since the `_helpers` module is already non-public, this function name can be public within the module